components/studentComboBox.vue

```diff
@@ -219,9 +220,13 @@ export default {
       token: state => state.auth.token
     }),
     displayStatus() {
-      if (this.studentData.isActive) {
-        return 'Ativo'
-      } else return 'Inativo'
+      return this.getStatus(
```

Comment:
Muitos parâmetro. É interessante passar como parâmetros nomeados.

https://simonsmith.io/destructuring-objects-as-function-parameters-in-es6

Reply:
Mudei já. Ficou elegante, mas, acho que isso pode gerar uma insegurança. Talvez não na nossa aplicação hoje, mas tipo, estou enviando mais informações a essa função do que o necessário...

Reply:
Se quiser excluir dados, pode pegar só o que for necessário